osu.Framework/Screens/ScreenExitEvent.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+namespace osu.Framework.Screens
+{
+    public class ScreenExitEvent : ScreenEvent
+    {
+        /// <summary>
+        /// The <see cref="IScreen"/> that will be resumed next.
+        /// </summary>
+        public IScreen Next;
+
+        /// <summary>
+        /// The final <see cref="IScreen"/> of an exit operation.
```

Comment:
```suggestion
        /// The final <see cref="IScreen"/> of this exit operation.
```
I think `this` reads better? Unsure, reading this at first glance made me question the difference between `Next` and `Destination`.

Maybe should explicitly clarify in `Destination` that it's different from `Next` when this is a multiple exiting operations (i.e. from `MakeCurrent`).